### PR TITLE
Fix a bug in time stepping method

### PR DESCRIPTION
**Pull request by Rombur**
Fix a bug reported by Vaibhav Palkar on the mailing where the time step for embedded Runge-Kutta methods was not increased correctly.

### CODE REVIEW

**Comment by tjhei** [comment]
Did you check if there tests that now fail? If not, would you mind adding a new one (maybe just running step-52?)

**Comment by Rombur** [comment]
I have added a test.

File: tests/base/time_stepping_02.cc [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2014 - 2015 by the deal.II authors
```
Comment (tjhei) [inline tests/base/time_stepping_02.cc]: 2016

**Comment by Rombur** [comment]
done

**Pull request merged**